R/combine_cleaning_logs.R

```diff
@@ -44,7 +44,8 @@
 
    print(names(list_of_log) |> glue::glue_collapse(" ,")  %>% glue::glue("List of element to combine- ", .))
 
-   output[["cleaning_log"]] <- dplyr::bind_rows(list_of_log_only)
+   output[["cleaning_log"]] <- dplyr::bind_rows(list_of_log_only)|> dplyr::mutate(change_type = NA_character_,
+                                                                                  new_value = NA_character_)
 
```

Comment:
It cannot bind if there are doubles and characters.
```
check_data <-  cleaningtools::cleaningtools_raw_data %>% 
  add_percentage_missing()
checkss <- cleaningtools::check_for_pii(df = check_data) %>% 
  cleaningtools::check_outliers(uuid_col_name = "X_uuid") %>% 
  check_others(uuid = "X_uuid",
               var_list = names(cleaningtools::cleaningtools_raw_data |> dplyr::select(ends_with("_other")) |> dplyr::select(-contains(".")))) %>%
  check_percentage_missing(uuid_var = "X_uuid") %>% 
  check_for_logical(uuid_var = "X_uuid", check_to_perform = "inc_employment_pension == tot_expenses", description = "income equals to expenses")

cannot_bind <- create_combined_log(list_of_log = checkss)
```

![image](https://github.com/impact-initiatives/cleaningtools/assets/79831444/5c234bbc-f4a7-4111-9785-034a4bd3d88a)

```
  #format with scientific = F instead of as.character and options(scipen = 999) 
  #scipen will change some global variables.
  #justify and trim for formating
to_combined <- checkss %>% 
  purrr::map(.f = ~dplyr::mutate(., across(.cols = everything(), .fns = ~format(., scientific = F, justify = "none", trim = T))))
combined_log <- create_combined_log(list_of_log = to_combined)

```